lib/SimpleSchema.js

```diff
@@ -591,6 +591,32 @@ class SimpleSchema {
     return def.label;
   }
 
+  /**
+   * Gets a field's property
+   *
+   * @param {String} [key] The schema key, specific or generic.
+   *   Omit this argument to get a dictionary of all labels.
+   * @param {String} [prop] Name of the property to get.
+   *
+   * @returns {any} The property value
+   */
+  get(key, prop) {
+    const def = this.getDefinition(key, ['type', prop]);
+
+    if (!def) return undefined;
+
+    if (schemaDefinitionOptions.includes(prop)) {
```

Comment:
Hey, thanks for for pointing this. I meant to use `Array.prototype.includes`, i see some other issues with this #33. To be sure could you check you node version here `node -v` and to `import 'babel-polyfill'` at the top of the file throwing an error?